app/src/main/java/org/openobservatory/ooniprobe/model/NetworkMeasurement.java

```diff
@@ -40,11 +42,13 @@ public static String getTestName(Context context, String name) {
                 return context.getString(R.string.http_invalid_request_line);
             case OONITests.HTTP_HEADER_FIELD_MANIPULATION:
                 return context.getString(R.string.http_header_field_manipulation);
+            case OONITests.NDT:
+                return context.getString(R.string.ndt);
             case OONITests.NDT_TEST:
                 return context.getString(R.string.ndt);
```

Comment:
Uhm, shouldn't you delete the case where `NDT_TEST` is used?

Reply:
This is needed because of old apps have ndt_test instead of ndt string

Reply:
Why is it needed? This is a new app

Reply:
Because the stored tests on the device can have the old test name

Reply:
LOL, I did not think at that! My bad! :-D